ideascube/serveradmin/management/commands/catalog.py

```diff
@@ -149,6 +150,14 @@ def list_packages(self, options):
                 for pkg in pkgs:
                     print(fmt.format(pkg))
 
+        pkgs = self.catalog.list_nothandled(options['ids'])
+
+        if pkgs:
+            print('Not handled packages')
```

Comment:
If we decide to display the list of not handled, puting them at the end make them the more visible, given that the terminal is autoscrolling.
Maybe we could just have a warning before the table like "Skipping unsupported content: id1, id2, di3" ?